heartbeat/monitors/util.go

```diff
@@ -53,16 +52,16 @@ func (s IPSettings) Network() string {
 	return ""
 }
 
-func MakeSimpleJob(name, typ string, f func() (common.MapStr, error)) Job {
-	return MakeJob(name, typ, func() (common.MapStr, []TaskRunner, error) {
+func MakeSimpleJob(jobName string, f func() (common.MapStr, error)) Job {
+	return MakeJob(jobName, func() (common.MapStr, []TaskRunner, error) {
 		event, err := f()
 		return event, nil, err
 	})
 }
 
-func MakeJob(name, typ string, f func() (common.MapStr, []TaskRunner, error)) Job {
-	return &funcJob{name, typ, funcTask{func() (common.MapStr, []TaskRunner, error) {
-		return annotated(time.Now(), typ, f).Run()
+func MakeJob(jobName string, f func() (common.MapStr, []TaskRunner, error)) Job {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function MakeJob should have comment or be unexported